doc/Commands/Fix.md

```diff
@@ -0,0 +1,39 @@
+# Fix
+
+The `fix` command attempts to automatically fix WDL files with specification incompatibilities.
+
+## Example
+
+```bash
+$ java -jar target/wdlTools.jar fix -O gatk-fixed https://raw.githubusercontent.com/gatk-workflows/gatk4-genome-processing-pipeline/1.3.0/WholeGenomeGermlineSingleSample.wdl
+```
+
+## Usage
+
+```commandline
+Usage: wdlTools fix [OPTIONS] <path|uri>
+Fix specification incompatibilities in WDL file and all its dependencies.
+
```

Comment:
can you describe which incompatibilities are fixed, and which ones are not (e.g. variables with same name appearing in multiple sections)?

Reply:
I added this to the documentation